vite.config.js

```diff
@@ -0,0 +1,9 @@
+// vite.config.js
```

Comment:
this file is duplicate of `lib/tools/components-package/vite.config.js` Is it possible to reuse? or is it needed here?